example/server/api/index.js

```diff
@@ -1,32 +1,22 @@
 const Router = require('express').Router;
-const environmentID = 'uCDQzKWgejrutqSYYsKWen';
-const flagsmith = require('../../../');
-const NodeCache = require('node-cache');
+const { Flagsmith } = require('../../../sdk');
 
-flagsmith.init({
-    environmentID
-    // this is an example of a user-defined cache
-    /*cache: new NodeCache({
-        stdTTL: 5
-    })*/
+const flagsmith = new Flagsmith({
+    environmentKey: 'some-key'
```

Comment:
If we are going to have a non-working key as default, we should just check its existence and throw out of the example if one is not provided.